_posts/2023-05-01-release-highlights.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "OpenDDS 3.24 Highlights"
+categories: news
+author: OpenDDS
+---
+
+# Transition to Unity Foundation
+
+[OpenDDS 3.24](https://github.com/OpenDDS/OpenDDS/releases/tag/DDS-3.24) marks the first release of OpenDDS not performed by [Object Computing, Inc. (OCI)](https://objectcomputing.com)!
+As announced in {% post_url 2023-03-17-transition-to-unity %}, the administration of the OpenDDS Foundation passed from OCI to Unity Foundation in March 2023.
+The relevant points from the transition are:
+
+* OpenDDS will remain open-source software.
+* The OpenDDS Foundation will continue to develop OpenDDS and support the OpenDDS Community.
+  OpenDDS Foundation will offer support contracts, training, consulting and open-source development related to OpenDDS.
+  OpenDDS Foundation will not develop any proprietary code.
+* OpenDDS Foundation has gained not-for-profit status.
+* Unity Foundation will dedicate one engineer to the maintenance, development, and release of OpenDDS.
+* Systems and processes administered by OCI on behalf of the OpenDDS Foundation will no longer be maintained.
+  Unity Foundation will take over some of these systems and processes while others will be deprecated and eliminated to reduce financial and mental overhead.
+
+For more on OCI's role, see {% link foundation/minutes/2023-04-12.html %}.
+
+As part of the transition, OpenDDS Foundation has adopted the following (tentative) calendar:
+
+| Month     | Activity                |
+|-----------|-------------------------|
+| January   | Release and TAB Meeting |
+| February  | Release Article         |
+| March     | Topic Article/Webinar   |
+| April     | Release and TAB Meeting |
+| May       | Release Article         |
+| June      | Townhall Webinar        |
+| July      | Release and TAB Meeting |
+| August    | Release Article         |
+| September | Topic Article/Webinar   |
+| October   | Release and TAB Meeting |
+| November  | Release Article         |
+| December  | Year in Review Webinar  |
+
+The general plan is to have one release and [Technical Advisory Board (TAB) Meeting]({% link foundation/index.html }%) every quarter.
+This will be followed by a news article (like this one) that goes over the highlights of current release.
+In June and December, there will be webinars to get feedback from the community and reflect on progress.
+In March and September, there will an article or webinar on some aspect of OpenDDS.
+
+# OpenDDS Developer's Guide
+
+A major accomplishment in OpenDDS 3.24 was the migration of the Developer's Guide from an Open Office Document that was held hostage on a private SVN server to [ReStructured Text in the repository](https://github.com/OpenDDS/OpenDDS/tree/master/docs/devguide).
+The Developer's Guide corresponding to the master branch is hosted at https://opendds.readthedocs.io/en/latest-release/.
+Versions for releases will also be available, e.g., https://opendds.readthedocs.io/en/dds-3.24/.
```

Comment:
The version names on RTD are basically the same as the tags and branches in git. If you really want to master, it's https://opendds.readthedocs.io/en/master/. There's a bit more to it we could add though:

```suggestion
The Developer's Guide corresponding to the latest version is hosted at https://opendds.readthedocs.io/en/latest-release/.
There is also a version that follows the latest development on the master branch at https://opendds.readthedocs.io/en/master.
Specific versions, e.g., https://opendds.readthedocs.io/en/dds-3.24/ can be viewed by using the Read the Docs version switcher (the book icon always at the bottom of the viewable part of the page), or by choosing from the versions on [the OpenDDS Read the Docs project page](https://readthedocs.org/projects/opendds/).
```